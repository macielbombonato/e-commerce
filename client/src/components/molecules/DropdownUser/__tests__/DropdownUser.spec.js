import createTestContext from 'react-cosmos-test/enzyme';

import fixture from '../DropdownUser.fixture';

const { mount, getWrapper } = createTestContext({ fixture });

beforeEach(mount);

describe('#DropdownUser', () =>
  it('renders component', () => {
    expect(getWrapper()).toMatchSnapshot();
  }));
