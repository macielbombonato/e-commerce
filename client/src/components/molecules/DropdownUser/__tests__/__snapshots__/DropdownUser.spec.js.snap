// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#DropdownUser renders component 1`] = `
<DropdownUser
  logout={[Function]}
  name="Name"
>
  <ButtonDropdown
    isOpen={false}
    toggle={[Function]}
  >
    <Dropdown
      dropup={false}
      group={true}
      isOpen={false}
      nav={false}
      toggle={[Function]}
    >
      <Manager
        className="btn-group"
        onKeyDown={[Function]}
        tag="div"
      >
        <div
          className="btn-group"
          onKeyDown={[Function]}
        >
          <DropdownToggle
            aria-haspopup={true}
            caret={true}
            color="info"
          >
            <Target
              aria-expanded={false}
              aria-haspopup={true}
              className="dropdown-toggle"
              color="info"
              component={[Function]}
              onClick={[Function]}
            >
              <Button
                aria-expanded={false}
                aria-haspopup={true}
                className="dropdown-toggle"
                color="info"
                innerRef={[Function]}
                onClick={[Function]}
                tag="button"
              >
                <button
                  aria-expanded={false}
                  aria-haspopup={true}
                  className="dropdown-toggle btn btn-info"
                  onClick={[Function]}
                  type="button"
                >
                  Name
                </button>
              </Button>
            </Target>
          </DropdownToggle>
          <DropdownMenu
            flip={true}
            tag="div"
          >
            <div
              aria-hidden={true}
              className="dropdown-menu"
              role="menu"
              tabIndex="-1"
            >
              <DropdownItem
                onClick={[Function]}
                tag="button"
                toggle={true}
              >
                <button
                  className="dropdown-item"
                  onClick={[Function]}
                  tabIndex="0"
                  type="button"
                >
                  Logout
                </button>
              </DropdownItem>
            </div>
          </DropdownMenu>
        </div>
      </Manager>
    </Dropdown>
  </ButtonDropdown>
</DropdownUser>
`;
